core/mapreduce/src/main/java/mil/nga/giat/geowave/mapreduce/output/GeoWaveOutputFormat.java

```diff
@@ -56,8 +58,32 @@
 					persistentAdapterStore.addAdapter(a);
 				}
 			}
+
 			final IndexStore persistentIndexStore = GeoWaveStoreFinder.createIndexStore(configOptions);
 			final Index[] indices = JobContextIndexStore.getIndices(context);
+			StringBuilder sbDebug = new StringBuilder();
```

Comment:
I think we should wrap this with if(LOGGER.isDebugEnabled()){<build string>} so we're not running through building this string in production